### DISCUSSION TITLE
新入软路由，想请教各位大佬如何搞定RSS的

### DISCUSSION THREAD

**Discussion by ghost** [istoreos/istoreos#1360] [Q&A] [answered]
新入软路由这个坑,听人建议刷了istoreos
自己捣鼓阶段是用的qbittorrent,但是发现占用有些高,想换aria或transmission
网上说用rss要装flexget,我在路由器里没有搜到这个软件
所以想请教下各位大佬是如何搞定rss的?

**Comment by bcseputetto** [comment, marked as answer]
如果要用 flexget 可以考虑用 docker 来跑
首先确保你的 iStoreOS 已经挂载了大容量分区，并且docker也已经迁移到大容量分区的挂载路径
![image](https://github.com/istoreos/istoreos/assets/29139573/53f98822-9f52-49b3-a92b-c31a81a7dbc2)
![image](https://github.com/istoreos/istoreos/assets/29139573/86662a43-4024-468e-b8b8-3b562ef4976c)
以我的 `mmcblk0p4` 分区4，挂载路径 `/mnt/mmc0-4` 为例
```console
mkdir -p /mnt/mmc0-4/flexget/config
#创建flexget的配置文件夹，把flexget的配置文件放进去，配置文件自己去网上找教程，配置文件名为 config.yml
docker run -d \
  --name flexget \
  -v /mnt/mmc0-4/flexget/config:/config \
  -e TZ="Asia/Shanghai" \
  -p 5050:5050 \
  --restart=unless-stopped \
  flexget/flexget:latest \
  daemon start --autoreload-config
#用docker cli 创建 flexget 容器的示例
#-v那行需要修改成你自己的路径
#-p那行是端口映射，是webui的端口，如果你的flexget配置文件里web_server的port并非5050，请你修改相应的端口，要么改配置文件的port为5050，要么把-p那行的端口修改成配置文件里的端口
```
![image](https://github.com/istoreos/istoreos/assets/29139573/ac6f9f61-4bfa-4b6f-96b9-ff0a901e375d)

**Reply by ghost** [reply]
感谢回复

之前用的是朋友淘汰送我“练手“的N3700。昨晚正好打游戏，被蛊惑+冲动🤡，下单了N305。。。
所以可以继续用QBT

感觉掉进了个大坑里🙃🙃🙃
